SignalServiceKit/src/Messages/TSGroupModel.m

```diff
@@ -21,6 +23,22 @@ - (instancetype)initWithTitle:(NSString *)title
     return self;
 }
 
+- (nullable instancetype)initWithCoder:(NSCoder *)coder
+{
+    self = [super initWithCoder:coder];
+    if (!self) {
+        return self;
+    }
+
+    // Occasionally seeing this as nil in legacy data,
+    // which causes crashes.
+    if (_groupMemberIds == nil) {
```

Comment:
The proximate cause of the crash is that, while searching we enumerate all member ids of all groups, and assume groupMemberIds is non-nil.

This change let's us maintain that invariant rather than making the property nullable and adding nil-checking in multiple places.